packages/protocol/src/Chief.sol

```diff
@@ -86,7 +89,7 @@ contract Chief is CoreRoles, AccessControl, IChief {
   function deployVault(
     address _factory,
     bytes calldata _deployData,
-    string calldata rating
+    uint256 rating
```

Comment:
Inconsistent naming, why do we use here "_name"? There are no global `factory` or `_deployData`

Reply:
Updated variable naming with no underscore since you are right; there is no variable collision.